e2e/aws/fixtures_test.go

```diff
@@ -138,9 +138,9 @@ func newTeleportConfig(t *testing.T) *servicecfg.Config {
 	// run multiple tests in parallel.
 	tconf.Console = nil
 	tconf.Proxy.DisableWebInterface = true
-	tconf.PollingPeriod = 500 * time.Millisecond
-	tconf.ClientTimeout = time.Second
-	tconf.ShutdownTimeout = 2 * tconf.ClientTimeout
+	tconf.Testing.PollingPeriod = 500 * time.Millisecond
+	tconf.Testing.ClientTimeout = time.Second
+	tconf.Testing.ShutdownTimeout = 2 * tconf.Testing.ClientTimeout
```

Comment:
It looks like these can actually be set by the Teleport service config, let's keep these out of `Testing`.

Reply:
As discussed offline, these are only set (overriden defaults) by the tests, and can't be set by the config file/flags.